File: sdk/go/provider/block_bdea_cc.go
package provider

// block.bdea.cc：mailinator 官方姊妹子域名，MX 指向 mail.mailinator.com。

func BlockBdeaCcGenerate() (*CreatedMailbox, error) {
	local := randomStr(12)
	return &CreatedMailbox{
		Channel: "block-bdea-cc",
		Email:   local + "@block.bdea.cc",
	}, nil
}

func BlockBdeaCcGetEmails(email string) ([]NormEmail, error) {
	return MailinatorGetEmails(email)
}

File: sdk/python/tempmail_sdk/providers/block_bdea_cc.py
"""Mailinator 姊妹域名 block.bdea.cc"""

from typing import List
from ..types import Email, EmailInfo
from . import mailinator

CHANNEL = "block-bdea-cc"
DOMAIN = "block.bdea.cc"


def generate_email() -> EmailInfo:
    local = mailinator._random_string(12)
    return EmailInfo(channel=CHANNEL, email=f"{local}@{DOMAIN}")


def get_emails(token: str, email: str = "", **kwargs) -> List[Email]:
    return mailinator.get_emails(token, email, **kwargs)
